.github/pull_request_template.md

```diff
@@ -0,0 +1,22 @@
+# Description
+Briefly describe the problem or reason for the changes implemented in the PR
+
+# Trac ticket
+Link to the accepted Trac ticket for this PR. All PRs except docs or tests changes need an accepted ticket. New features need community consensus.
```

Comment:
I think a new feature ticket would need to already have community consensus to be an accepted ticket, so that bit seems a bit redundant to me.

Reply:
_"All PRs except docs or tests changes need an accepted ticket."_

This sentence is not true and really misleading, check out [docs](https://docs.djangoproject.com/en/5.0/internals/contributing/writing-code/submitting-patches/#typo-fixes-and-trivial-documentation-changes). For example, small cleanups in the source code may not require a ticket, and new pages in docs may required a ticket.